### ISSUE TITLE
Handle more inputs with argparse

### ISSUE THREAD

**Issue by mobiuscreek** [BritishGeologicalSurvey/CoreScore#5]
The training script, `train.py` only accepts the number of epochs and learning rate. The default batch size is set t to 1 and as a result the training has been very slow.  It will be good to be able handle more arguments and see how they affect performance. 

## Acceptance criteria:

- [ ] Batch size
- [ ] Weight decay
- [ ] Path
